src/aatoolbox/datasources/cds/area.py

```diff
@@ -32,83 +31,31 @@
     ----------
-    north : float
-        The northern latitude boundary of the area (degrees)
-    south : float
-        The southern latitude boundary of the area (degrees)
-    east : float
-        The easternmost longitude boundary of the area (degrees)
-    west : float
-        The westernmost longitude boundary of the area (degrees)
+    round_coords: bool, default = False
+        Whether or not to round the coordinates. In general this should be
+        done, but this option has been left in for some legacy data.
+    offset_val: float, default = 0.0
+        Offset the coordinates by this factor. Some CDS datasets require
+        coordinates that end in 0.5.
+
+    Returns
+    -------
+    List of coordinates in the correct order for the API (north, west,
+    south, east)
     """
-
-    def __init__(self, north: float, south: float, east: float, west: float):
-        self.north = north
-        self.south = south
-        self.east = east
-        self.west = west
-
-    def list_for_api(
-        self,
-        round_coords: bool = True,
-        offset_val: float = 0.0,
-    ) -> List[float]:
-        """
-        List the coordinates in the order that they're needed for the API.
-
-        Parameters
-        ----------
-        round_coords: bool, default = False
-            Whether or not to round the coordinates. In general this should be
-            done, but this option has been left in for some legacy data.
-        offset_val: float, default = 0.0
-            Offset the coordinates by this factor. Some CDS datasets require
-            coordinates that end in 0.5.
-
-        Returns
-        -------
-        List of coordinates in the correct order for the API (north, west,
-        south, east)
-        """
-        if not round_coords:
-            return [self.north, self.west, self.south, self.east]
-
-        return [
-            self._round_coord(
-                coord=getattr(self, coordinate),
-                direction=direction,
-                offset_val=offset_val,
-            )
-            for coordinate, direction in {
-                "north": "up",
-                "west": "down",
-                "south": "down",
-                "east": "up",
-            }.items()
-        ]
-
-    @staticmethod
-    def _round_coord(
-        coord: float,
-        direction: str,
-        offset_val: float = 0.0,
-        round_val: int = 1,
-    ) -> float:
-        if direction == "up":
-            function = np.ceil.__call__  # needed for mypy
-            offset_factor = 1
-        elif direction == "down":
-            function = np.floor.__call__  # needed for mypy
-            offset_factor = -1
-        return (
-            function(coord / round_val) * round_val
-            + offset_factor * offset_val
-        )
+    if round_coords:
+        area.round_area_coords(offset_val=offset_val)
+    return [area.north, area.west, area.south, area.east]
 
 
 class AreaFromStations(Area):
```

Comment:
Since there is only one subclass, I would just make this the `__init__` of `Area` and only have one class. Also maybe we should think about changing the name to make it more descriptive? `Area` is so general. Maybe `GeographicArea`?

Reply:
As discussed, only using the `utils/area.py` class now
I would agree `Area` is a bit general. I am now also thinking that it is more the bounding box than the area. Would `GeographicBoundingBox` make sense? Or too long?